build_tools/cmake/iree_native_test.cmake

```diff
@@ -113,6 +113,25 @@ function(iree_native_test)
         "TEST_TMPDIR=${_ANDROID_ABS_DIR}/test_tmpdir"
     )
     set_property(TEST ${_TEST_NAME} PROPERTY ENVIRONMENT ${_ENVIRONMENT_VARS})
+  elseif(CMAKE_SYSTEM_PROCESSOR STREQUAL "riscv" AND RISCV_CPU STREQUAL "rv64")
```

Comment:
> I'd love to see us move to something where we have a "run configuration" for tests that gets set in one place instead of having to do shenanigans in every test rule. Somewhat similar to Bazel's --run_under. We're not there yet though

Agreed. I might take a look at that if I find myself adding web browser / Emscripten stuff to these files too.